cmake/compiler/gcc/target_warnings.cmake

```diff
@@ -82,9 +82,11 @@ endmacro()
 
 macro(toolchain_cc_warning_extended)
 
+if(CMAKE_CXX_COMPILER_VERSION VERSION_GREATER "4.6.0")
```

Comment:
Could we use CMAKE_C_COMPILER_VERSION instead? In case a toolchain is built without CXX support.